deployments/mainnet/weth/migrations/1731429132_add_meth_collateral.ts

```diff
@@ -0,0 +1,133 @@
+import { expect } from 'chai';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { exp, proposal } from '../../../../src/deploy';
+
+const METH_ADDRESS = '0xd5F7838F5C461fefF7FE49ea5ebaF7728bB0ADfa';
+const METH_EXCHANGE_RATE_PROVIDER_ADDRESS = '0xe3cBd06D7dadB3F4e6557bAb7EdD924CD1489E8f';
```

Comment:
Why is this address used. Chainlink suggests another: 0x5b563107C8666d2142C216114228443B94152362. Consider replacing or explain why this address is used.